### PR TITLE
feat(Google Cloud): Add Google Cloud Audit logs, Google Kubernetes Engine, Google VPC Flow Logs.

### PR DESCRIPTION
**Pull request by PierrickV**
Doc for:
- Google Cloud audit logs including Google Workspace
- Google Kubernetes Engine
- Google VPC Flow Logs

### CODE REVIEW

**Review by squioc — APPROVED** [review]

File: docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{!operation_center/integration_catalog/cloud_and_saas/google/google_cloud.md!}
+
+{!operation_center/integration_catalog/generated/google-kubernetes-engine_do_not_edit_manually.md!}
+
+
+## Google Kubernetes Engine (GKE) configuration
+
+There's multiple kind of logs produced by GKE:
+
+**Auditd logs**: Most import logs from a security point of view. We recommend that you use [Auditbeat](../../../endpoint/auditbeat_linux/) to collect Auditd logs.
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md:14]:
```suggestion
**Auditd logs**: Most important logs from a security point of view. We recommend that you use [Auditbeat](../../../endpoint/auditbeat_linux/) to collect Auditd logs.
```

File: docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Flow Logs**: From [Google VPC FLow Logs documentation](https://cloud.google.com/vpc/docs/using-flow-logs):
+> VPC Flow Logs records a sample of network flows sent from and received by VM instances, including instances used as GKE nodes. These logs can be used for network monitoring, forensics, real-time security analysis, and expense optimization.
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md:17]:
```suggestion
> VPC Flow Logs records a sample of network flows sent from and received by VM instances, including instances used as GKE nodes. These logs can be used for network monitoring, forensics, real-time security analysis and expense optimization.
```

File: docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Please read the [dedicated documentation](https://docs.sekoia.io/integrations/integrations/google_vpc_flow_logs.md).
+(*Intake type: Google VPC Flow Logs*)
+
+**Activity logs** (*Intake type: Google Cloud Audit log*):
+
+  - operations done on GCP cluster resources (create a cluster, etc.) `resource.type: gke_cluster`
+  - operations done on Kubernetes objects “inside” clusters (Kubernetes Audit Policy) `resource.type: k8s_cluster`
+
+**Event logs**: Kubernetes events are objects that provide insight into what is happening inside a cluster, such as what decisions were made by scheduler or why some pods were evicted from the node `log_id("events")` (*Intake type: Google Cloud Audit log*)
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md:27]:
```suggestion
**Event logs**: Kubernetes events are objects that provide insight into what is happening inside a cluster, such as what decisions were made by the scheduler or why some pods were evicted from the node `log_id("events")` (*Intake type: Google Cloud Audit log*)
```

File: docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+**Node logs**: Services running on GKE nodes (kubelet, node problem detector, container runtime, etc.) emit their own logs `resource.type: k8s_node`
+
+**Container logs**: Container stdout and stderr streams `log_id("stdout") log_id("stderr")`
+
+**Instance logs**: Serial console output `log_id("serialconsole.googleapis.com/serial_port_1_output")`
+
+To go further, consult the following resources:
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md:35]:
```suggestion
To learn more, consult the following resources:
```

File: docs/operation_center/integration_catalog/cloud_and_saas/google/google_workspace.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+{!operation_center/integration_catalog/cloud_and_saas/google/google_cloud.md!}
+
+{!operation_center/integration_catalog/generated/google-cloud-audit_do_not_edit_manually.md!}
+
+## Google Workspace configuration
+
+To begin, follow the official documentation to [share audit logs with Google Cloud](https://cloud.google.com/logging/docs/audit/configure-gsuite-audit-logs). 
+
+Once log sharing activated, you should be able to see Google Workspace events in Google Cloud Log Explorer.
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/cloud_and_saas/google/google_workspace.md:13]:
```suggestion
Once log sharing is activated, you should be able to see Google Workspace events in Google Cloud Log Explorer.
```

File: docs/operation_center/integration_catalog/generated/google-kubernetes-engine_do_not_edit_manually.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+| Data Source | Description                          |
+| ----------- | ------------------------------------ |
+| `Application logs` | Google Kubernetes Engine logs contains events from applications deployed in Google Cloud containers. |
+
+
+
+
+
+In details, the following Table denotes the type of events produced by this integration.
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/generated/google-kubernetes-engine_do_not_edit_manually.md:15]:
```suggestion
In details, the following table denotes the type of events produced by this integration.
```

File: docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md (hunk 2 above)
Comment (PierrickV) [reply docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md:17]: This is a quote from Google's documentation

**Review by squioc — APPROVED** [review]

File: docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+{!operation_center/integration_catalog/cloud_and_saas/google/google_cloud.md!}
+
+{!operation_center/integration_catalog/generated/google-kubernetes-engine_do_not_edit_manually.md!}
+
+
+## Google Kubernetes Engine (GKE) configuration
+
+There are different types of logs produced by GKE:
+
+**Auditd logs**: Most important logs from a security point of view. We recommend that you use [Auditbeat](../../../endpoint/auditbeat_linux/) to collect Auditd logs.
```
[the remark this answers is not in the document]
Comment (ka0ula) [reply docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md:14]: It depends on the alternatives. Is it possible to collect these logs using another tool? 
If you want the user to use Auditbeat only, it's better to say "We recommend that you use"

[the remark this answers is not in the document]
Comment (PierrickV) [reply docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md:14]: > We recommend that you use

Seems to be appropriate for what we want to say thank you @squioc and @ka0ula

**Pull request merged**